hw/ip/adc_ctrl/doc/dv/index.md

```diff
@@ -52,20 +52,14 @@ ADC_CTRL testbench instantiates (already handled in CIP base env) [tl_agent]({{<
 which provides the ability to drive and independently monitor random traffic via
 TL host interface into ADC_CTRL device.
 
-### UVC/agent 1
-[Describe here or add link to its README]
-
-### UVC/agent 2
-[Describe here or add link to its README]
+### push_pull agents
+The ADC data interface is emulated using push_pull agents. ([`push_pull_agent`]({{< relref "hw/dv/sv/push_pull_agent/README.md" >}})
```

Comment:
we might have to take a closer look at this later. I don't believe the adc's behavior is identical to that of a push pull.
We tried to ask for a change, but the feature was already in.

Reply:
@tjaychen We have updated this agent to support four-phase handshake, so adc_ctrl can use it directly

Reply:
ahh okay. thank you for clarifying.